lima-init.sh

```diff
@@ -55,6 +55,8 @@ flag {
 }
 EOF
 echo "#LIMA-END" >> /etc/fstab
+mkmntdirs
+mount -a
```

Comment:
These two lines prevented lima from starting for me, not until they were executed as part of provision scripts. I noticed that during my tests since the 9p PR got merged into Lima.

Does it work fine for RD?

Reply:
> These two lines prevented lima from starting for me, not until they were executed as part of provision scripts.

They only work with the latest lima `master` that has https://github.com/lima-vm/lima/pull/843 merged.

That change was also necessary to get the `ci` edition working because cloud-init was running into the same problem.

> Does it work fine for RD?

No. It will also require a Lima update before it can use a new image.

Note that I haven't done a release yet; I hope to get some time to implement #54 because the new certs feature isn't supported by lima-init yet either.

Reply:
I'm not sure if I answered your question correctly.

These 2 lines are necessary to actually get the mounts to be, well mounted. And they work fine in a provision script.

They didn't work as part of lima-init because they happened too early, which is why I made the change in lima to unmount the `/tmp/*` mountpoints again, so that on a fresh install we can still move the `/tmp` dir to the data volume.

This may seem slightly too complex, but the reason I did it this way is to make it work with both cloud-init and lima-init.

The move to the data volume is not because we want to persist `/tmp`, but because we don't want to keep it in RAM because then creating large files in `/tmp` can eat up memory that we rather want to use for running containers. 😄

Reply:
Yeah, you answered my questions. Thanks :)

I was only trying to ascertain if the behaviour is same for others as well.